crates/biome_cli/Cargo.toml

```diff
@@ -43,6 +43,7 @@ dashmap                  = { workspace = true }
 hdrhistogram             = { version = "7.5.4", default-features = false }
 indexmap                 = { workspace = true }
 lazy_static              = { workspace = true }
+quick-junit              = "0.4.0"
```

Comment:
Should this be on the root cargo.toml, and have `workspace = true`? Although, I'm ok with here as well, since I assume this crate will logically always belong to `biome_cli`.

Reply:
Yeah, it only belongs to `biome_cli` so there's so need to make it available to the entire workspace